<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>921</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>921</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №5</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Алгоритм заполнения с упорядоченным&lt;br/&gt;списком ребер (с использованием списка&lt;br/&gt;активных ребер)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>175</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>175</height>
         </size>
        </property>
        <property name="title">
         <string>Цвет</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Цвет рисования</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Цвет фона</string>
         </property>
        </widget>
        <widget class="QPushButton" name="changePenColorBtn">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
        <widget class="QPushButton" name="changeBackgroundColorBtn">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>130</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
        <widget class="QLabel" name="PenColor">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: black</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="backgroundColor">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: white</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="paintShapeBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>460</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Закрасить фигуру</string>
         </property>
        </widget>
        <widget class="QPushButton" name="closeShapeBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>330</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Замкнуть фигуру</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>560</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Очистить плоскость</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Время выполнения</string>
         </property>
        </widget>
        <widget class="QLabel" name="timeLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>211</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>0 мс</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>430</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>С задержкой</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action">
   <property name="text">
    <string>Загрузить данные</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
